netcore/Startup.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Microsoft.AspNetCore.Mvc.Versioning;
+using Microsoft.AspNetCore.Mvc.ApiExplorer;
+using Swashbuckle.AspNetCore.Swagger;
+using OrderCaptureAPI.Util;
+
+namespace OrderCaptureAPI
+{
+    public class Startup
+    {
+        public Startup(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
+        public IConfiguration Configuration { get; }
+
+        // This method gets called by the runtime. Use this method to add services to the container.
+        public void ConfigureServices(IServiceCollection services)
+        {
+            var insightsKey = System.Environment.GetEnvironmentVariable("APPINSIGHTS_KEY");

```

Comment:
I found it useful to be able to view my own Application Insights telemetry by providing my own key. Can we send the team tracking telemetry using a different client?

Reply:
We can use two different ENV for a private and public App insights key. We do not expect attendees to have to change code but clearly they can if they want to